### PR TITLE
Remove duplicate information about the Google Cloud Storage Loader

### PR DESCRIPTION
**Pull request by lmath**
Currently, in our manual setup guides, we tend to point the reader to pertinent information in the Pipeline Components and Applications section at the first opportunity. For the GCS loader, we had duplicate information in the Pipeline Components and Applications section and the GCP Manual Setup section, so here we remove the duplicate info.

### CODE REVIEW

File: docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ date: "2020-03-02"
...
+Snowplow supports streaming data (either raw or enriched data) from [Pub/Sub](https://cloud.google.com/pubsub/) into [Google Cloud Storage (GCS)](https://cloud.google.com/storage/) with the [Google Cloud Storage Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md). 
 
-In addition, we recommend streaming the "failed events" Pub/Sub topic (containing data that fails to be processed successfully) to GCS, where it is possible to query the failed events to understand why they were not successfully processed and recover them using [Snowplow Event Recovery](https://github.com/snowplow-incubator/snowplow-event-recovery).
+We recommend streaming the "failed events" Pub/Sub topic (containing data that failed to be processed successfully) to GCS. From there, you may wish to load them into your destination of choice to query the failed events to understand why they were not successfully processed and recover them using [Snowplow Event Recovery](https://github.com/snowplow-incubator/snowplow-event-recovery).
```
Comment (stanch) [inline docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md:9]: “query failed events” should link to the page about querying them in BigQuery (https://docs.snowplow.io/docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/)

File: docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ date: "2020-03-02"
...
 
-To stream data from any of the Snowplow Pub/Sub topics (i.e. raw or enriched) to GCS, you need to use the [Google Cloud Storage Loader](https://github.com/snowplow-incubator/snowplow-google-cloud-storage-loader).
+You may also wish to stream the "good events" Pub/Sub topic to serve as a backup in case there is an issue downstream (e.g. while loading the data into BigQuery).  
 
-Cloud Storage Loader is a [Dataflow](https://cloud.google.com/dataflow) job which dumps event from an input [PubSub](https://cloud.google.com/pubsub/) subscription into a [Cloud Storage](https://cloud.google.com/storage/) bucket.
+If you need to load events from GCS into another destination, you'll need to follow the directions for your favoured destination to load in data from GCS.
```
Comment (stanch) [inline docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md:13]:
```suggestion
If you need to load events from GCS into another destination, you'll need to follow the directions for your favored destination on loading data from GCS. 
```
We favor American spelling :)

File: docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md [hunk 3]
```diff
@@ -182,12 +160,4 @@ To display documentation about Cloud Storage Loader-specific options:
 
 A full list of all the Beam CLI options can be found at: [https://cloud.google.com/dataflow/pipelines/specifying-exec-params#setting-other-cloud-pipeline-options](https://cloud.google.com/dataflow/pipelines/specifying-exec-params#setting-other-cloud-pipeline-options).
 
-### Tests and debugging
```
[the remark this answers is not in the document]
Comment (stanch) [reply docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md:185]: Agree

File: docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md [hunk 4]
```diff
@@ -14,63 +14,38 @@ import CodeBlock from '@theme/CodeBlock';
...
-- `--shardTemplate=String` Default: -W-P-SSSSS-of-NNNNN The shard template which will be part of the filenames
-- `--outputFilenameSuffix=String` Default: .txt The suffix of the filenames written out
-- `--windowDuration=Int` Default: 5 The window duration in minutes
-- `--compression=String` Default: none The compression used (gzip, bz2 or none), bz2 can't be loaded into BigQuery
-- `--numShards=int` Default: 1 The maximum number of output shards produced when writing
+- `--inputSubscription=String` The Cloud Pub/Sub subscription to read from, formatted like projects/[PROJECT]/subscriptions/[SUB]. Required. 
+- `--outputDirectory=gs://[BUCKET]/` The Cloud Storage directory to output files to, ending in /. Required.  
+- `--outputFilenamePrefix=String` The prefix for output files. Default: output. Optional.
+- `--shardTemplate=String` A valid shard template as described [here](https://javadoc.io/static/com.google.cloud.dataflow/google-cloud-dataflow-java-sdk-all/1.7.0/com/google/cloud/dataflow/sdk/io/ShardNameTemplate.html), which will be part of the filenames. Default: -W-P-SSSSS-of-NNNNN. Optional.
```
Comment (stanch) [inline docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md:28]:
```suggestion
- `--shardTemplate=String` A valid shard template as described [here](https://javadoc.io/static/com.google.cloud.dataflow/google-cloud-dataflow-java-sdk-all/1.7.0/com/google/cloud/dataflow/sdk/io/ShardNameTemplate.html), which will be part of the filenames. Default: `-W-P-SSSSS-of-NNNNN`. Optional.
```

File: docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md [hunk 5]
```diff
@@ -14,63 +14,38 @@ import CodeBlock from '@theme/CodeBlock';
...
 
-## Technical details
-
 Cloud Storage loader is built on top of [Apache Beam](https://beam.apache.org/) and its Scala wrapper [SCIO](https://github.com/spotify/scio).
 
-It groups messages from an input PubSub subscription into configurable windows and write them out to Google Cloud Storage.
-
-It also optionally partitions the output by date so that you can easily see what was outputted when in your cloud storage bucket, for example we could see the following hierarchy:
-
-```bash
```
Comment (stanch) [inline docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md:25]: This particular example seems useful, could we include it as an illustration of the partitioning configuration option below?

File: docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md [hunk 6]
```diff
@@ -14,63 +14,38 @@ import CodeBlock from '@theme/CodeBlock';
...
+- `--numShards=Int` The maximum number of output shards produced when writing. Default: 1. Optional.
+- `--dateFormat=YYYY/MM/dd/HH/`  A date format string used for partitioning via date in `outputDirectory` and `partitionedOutputDirectory`. Default: `YYYY/MM/dd/HH/`. Optional.
+- `--partitionedOutputDirectory=gs://[BUCKET]/` The Cloud Storage directory to output files to, partitioned by schema, ending with /. Unpartitioned data will be sent to `outputDirectory`. Optional.
+
+
 
 #### Dataflow options
 
-To run on Dataflow, Beam Enrich will rely on a set of additional configuration options:
+To run on Dataflow, it is also necessary to specify additional configuration options. None of these options have default values, and they are all required.
```
Comment (stanch) [inline docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md:40]:
```suggestion
To run the Cloud Storage Loader on Dataflow, it is also necessary to specify additional configuration options. None of these options have default values, and they are all required.
```

**Review by stanch — CHANGES_REQUESTED** [review]

File: docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md (hunk 2 above)
Comment (lmath) [reply docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md:13]: I didn't ~realise~ realize! 🇺🇸

File: docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md [hunk 7]
```diff
@@ -4,10 +4,12 @@ date: "2020-03-02"
...
+Snowplow supports streaming data (either raw or enriched data) from [Pub/Sub](https://cloud.google.com/pubsub/) into [Google Cloud Storage (GCS)](https://cloud.google.com/storage/) with the [Google Cloud Storage Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/google-cloud-storage-loader/index.md). 
 
-In addition, we recommend streaming the "failed events" Pub/Sub topic (containing data that fails to be processed successfully) to GCS, where it is possible to query the failed events to understand why they were not successfully processed and recover them using [Snowplow Event Recovery](https://github.com/snowplow-incubator/snowplow-event-recovery).
+We recommend streaming the "failed events" Pub/Sub topic (containing data that failed to be processed successfully) to GCS. From there, you may wish to load them into your destination of choice to [query the failed events](/docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/) to understand why they were not successfully processed and recover them using [Snowplow Event Recovery](https://github.com/snowplow-incubator/snowplow-event-recovery).
```
Comment (stanch) [inline docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/index.md:9]:
```suggestion
We recommend streaming the "failed events" Pub/Sub topic (containing data that failed to be processed successfully) to GCS. From there, you may wish to load them into your destination of choice to [query the failed events](/docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md) to understand why they were not successfully processed and recover them using [Snowplow Event Recovery](https://github.com/snowplow-incubator/snowplow-event-recovery).
```

**Review by stanch — CHANGES_REQUESTED** [review]

File: static/_redirects [hunk 8]
```diff
@@ -56,3 +56,6 @@
 
 /docs/open-source-quick-start/quick-start-installation-guide-on-gcp/summary-of-what-you-have-deployed-gcp/ /docs/open-source-quick-start/quick-start-installation-guide-on-gcp/summary-of-what-you-have-deployed/ 301
 /docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-postgres-data/ /docs/getting-started-on-snowplow-open-source/quick-start-gcp/query-your-data/ 301
+
+/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/downloading-google-cloud-storage-loader/ /getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-destinations/google-cloud-storage/ 301
```
Comment (stanch) [inline static/_redirects:60]: Something is off here:
* Does `setup-destinations/google-cloud-storage/` exist in this PR?
* The redirect location should start with `/docs`

File: static/_redirects [hunk 9]
```diff
@@ -56,3 +56,6 @@
...
+/docs/getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-google-cloud-storage-gcs-destination/configuration-options/ /getting-started-on-snowplow-open-source/setup-snowplow-on-gcp/setup-destinations/google-cloud-storage/ 301
```
Comment (stanch) [inline static/_redirects:61]: Ditto

File: static/_redirects (hunk 8 above)
Comment (lmath) [reply static/_redirects]: Sorry about that. I've updated and tested them now.

**Pull request merged**